views/js/pciManager/tpl/listing.tpl

```diff
@@ -1,12 +1,17 @@
+    <div class="action-header">
+        <span class="switch-header">Enabled in item authoring</span>
+    </div>
 {{#each interactions}}
 <li data-type-identifier="{{typeIdentifier}}" class="pci-list-element{{#unless enabled}} pci-disabled"{{/unless}}">
     <span class="desc truncate">
-        <span class="name">({{model}}) - {{label}} - v{{version}}</span>
+        <span class="name">({{model}}{{#runtimeOnly}} <span class="runtime">- runtime only</span>{{/runtimeOnly}}) - {{label}} - v{{version}}</span>
```

Comment:
Is `{{#runtimeOnly}}` an undocumented feature of Handlebars?
I'd prefer to stick to the convention of `{{#if runtimeOnly}} ... {{/if}}`